cachet-tools/services.py

```diff
@@ -0,0 +1,53 @@
+import json
+import logging
+import pprint
+
+# format the logger message
+logging.basicConfig(level='INFO', format='%(asctime)s %(message)s', datefmt='%m/%d/%Y %H:%M:%S %p: ')
+logger = logging.getLogger(__name__)
+
+pp = pprint.PrettyPrinter(indent=4, compact=True)
+
+class Services:
+    'class for defining hosted services in Insights'
+    groupcount   = 0
+    serviceCount = 0
+
+    def __init__(self):
+        with open('/hosted-services.json') as services_file:
+            self.data = json.load(services_file)
+            Services.groupcount = len(self.data)
+            print("Group count: " + str(Services.groupcount))
+            
+    # end of init
+
+    def getData(self):
```

Comment:
this is probably a nit, but could you please change the naming style from camel case to underbar?  e.g. `getData` should be `get_data`, and `getServiceNameByLabel` should be `get_service_name_by_label`.

Reply:
Done!